src/components/manage/Blocks/Block/EditBlockWrapper.jsx

```diff
@@ -31,17 +34,16 @@ const EditBlockWrapper = (props) => {
     ? data.required
     : includes(config.blocks.requiredBlocks, type);
 
+  const styles = buildStyleClassNamesFromData(data.styles);
+
   return (
     <div
       ref={draginfo.innerRef}
       {...draginfo.draggableProps}
       // Right now, we can have the alignment information in the styles property or in the
       // block data root, we inject the classname here for having control over the whole
       // Block Edit wrapper
-      className={cx(`block-editor-${data['@type']}`, {
-        [data.align]: data.align,
```

Comment:
@sneridagh  I understand the desire to unpack all of the styles from data.styles but I don't understand the removal of data.align from the  block-editor classes in the idea that if  someone will not enable the styling for that block they will not get the styling align and now it's own align field  classes will not be present.